### PR TITLE
Internal review of cmake and github actions update

### PR DESCRIPTION
**Pull request by nbanerje**
* update to cmake and github actions
* Removed ifdef for protocol search as well

TODO:
@Marcus10110 when you have a chance I am not sure why the simulator does not produce protocol results. Data is definitely being generated, but it appears the analyzer thinks it isn't valid.
Screen Shot 2022-03-23 at 2 26 55 PM

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (Marcus10110) [inline .vscode/settings.json:1]: @nbanerje let's not commit the .vscode folder.

**Review by Marcus10110 — CHANGES_REQUESTED** [review]
2 things:
1. delete the .vscode folder. (don't add it to gitignore though)
2. delete the existing 'readme.saleae.md' file, since that's now outdated.

File: .vscode/settings.json (hunk 1 above)
Comment (nbanerje) [reply .vscode/settings.json:1]: oof how did that sneek in there! should we add it to .gitignore?

**Comment by Marcus10110** [comment]
Closing this, since this has been merged upstream.
@nbanerje I suggest we delete the entire forked repo on github once we're done, to reduce community confusion over what's the best fork to use.

**Pull request closed without merging**